src/algokit/cli/explore.py

```diff
@@ -98,4 +115,9 @@ def get_explore_url(network: str) -> str:
 def explore_command(network: str) -> None:
     url = get_explore_url(network)
     logger.info(f"Opening {network} explorer in your default browser")
+    if _is_wsl():
+        logger.warning(
```

Comment:
Alternatively (or even additionally) we could render the link that we tried to open, so the user can simply click it.